src/Infrastructure/Adapter/BdTopoRoadGeocoder.php

```diff
@@ -177,4 +177,39 @@ public function computeReferencePoint(
             return Coordinates::fromLonLat($coordinates[0], $coordinates[1]);
         }
     }
+
+    public function findRoadNames(string $search, string $cityCode): array
+    {
+        $filteredSearch = str_replace(' ', ' & ', trim($search));
```

Comment:
Je mettrais un commentaire avec un lien vers la doc de la syntaxe de tsquery de Postgres

En construisant toute la query ici on peut la filer en paramètre Doctrine

```suggestion
        // Build search query
        // https://www.postgresql.org/docs/current/datatype-textsearch.html#DATATYPE-TSQUERY
        $query = \str_replace(' ', ' & ', $search) . ':*';
```